src/clj/rems/db/applications.clj

```diff
@@ -91,7 +91,7 @@
    :get-form-template form/get-form-template
    :get-catalogue-item catalogue/get-localized-catalogue-item
    :get-license licenses/get-license
-   :get-user users/get-user-attributes
+   :get-user users/get-user
```

Comment:
Should `get-user-attributes` be removed or made private, if we don't want to use it?

Reply:
Done!